### PR TITLE
Wildfly14 new flow noamq

### PR DESCRIPTION
**Pull request by ebrygg**

### CODE REVIEW

**Comment by ebrygg** [comment]
test me

**Comment by ebrygg** [comment]
test me

File: wildfly-base/src/main/docker/Dockerfile [hunk 1]
```diff
@@ -41,10 +41,6 @@ COPY dependencies/jts-${wildfly.hibernate.module.jts.version}.jar $HIBERNATE_MOD
 COPY dependencies/geolatte-geom-${wildfly.hibernate.module.geolatte.version}.jar $HIBERNATE_MODULE
 COPY dependencies/hibernate-spatial-4x-to-5x-wrapper-${project.version}.jar $HIBERNATE_MODULE
 
-ENV YASSON_MODULE $JBOSS_HOME/modules/system/layers/base/org/eclipse/yasson/main
```
Comment (peerik) [inline wildfly-base/src/main/docker/Dockerfile]: I think this might cause problems for asset client

Comment (ebrygg) [reply wildfly-base/src/main/docker/Dockerfile]: oops

**Review by thomasfanto — APPROVED** [review]

**Pull request merged**